apps/api/src/app/integrations/usecases/get-webhook-support-status/get-webhook-support-status.usecase.ts

```diff
@@ -36,15 +35,15 @@ export class GetWebhookSupportStatus {
 
     return true;
   }
-  private async getIntegration(command: GetWebhookSupportStatusCommand): Promise<IntegrationEntity> {
+  private async getIntegration(command: GetWebhookSupportStatusCommand) {
```

Comment:
In this PR you were removing some of the return types for certain methods. What's the goal behind that? I can't find any pattern in those changes so I could understand why it is needed.

Reply:
the return type is inferred, in this case it's `IntegrationEntity | null`